lib/base.js

```diff
@@ -985,8 +985,17 @@ class Request extends EventEmitter {
 
     for (let index = 0; index < values.length; index++) {
       let value = values[index]
-      this.input(`param${index + 1}`, value)
-      command.push(`@param${index + 1}`, strings[index + 1])
+      if (Array.isArray(value)) {//if value is an array, prepare each items and add a coma between each item
```

Comment:
Instead of checking for an array, should we just check that the value is [iterable?](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/iterator)